documentation/book/proc-deploying-kafka-cluster-kubernetes.adoc

```diff
@@ -26,5 +28,5 @@ kubectl apply -f examples/kafka/kafka-persistent.yaml
 ----
 
 .Additional resources
-* For more information about deploying the Cluster Operator, see xref:cluster-operator-str[]
+* For more information about deploying the Cluster Operator, see xref:cluster-operator-{context}[]
```

Comment:
Replace {context} with 'str' which will be the default.

Reply:
This got skipped.

Reply:
This isn't one of the TO procs. I can fix it if you want, but I might just be creating conficts for @dmuntima.

Reply:
Yes, please fix it ;)

Reply:
@dmuntima has discovered that there are no defined procedures in the CO chapter so she's creating a list of modules that will need to created by engineering for this, so it won't effect her work.